test/test-template-engine.js

```diff
@@ -532,6 +532,36 @@ describe('html updaters', () => {
     container.remove();
   });
 
+  it('repeat re-runs each time', () => {
+    const getTemplate = ({ items, lookup }) => {
+      return html`
+      <div>
+        <ul id="target">
+          ${repeat(items, item => item.id, item => {
+            return html`<li id="${ item.id }">${ lookup?.[item.id] }</li>`;
```

Comment:
super-nit: I think proximal code doesn’t put spaces around interpolations:

```js
return html`<li id="${item.id}">${lookup?.[item.id]}</li>`;
```

(same goes for the other test)